enginetest/queries/query_plans.go

```diff
@@ -553,19 +553,21 @@ var PlanTests = []QueryPlanTest{
 	},
 	{
 		Query: `SELECT * FROM MYTABLE JOIN OTHERTABLE ON i = i2 AND s > s2`,
-		ExpectedPlan: "InnerJoin((mytable.i = othertable.i2) AND (mytable.s > othertable.s2))\n" +
+		ExpectedPlan: "IndexedJoin((mytable.i = othertable.i2) AND (mytable.s > othertable.s2))\n" +
 			" ├─ Table(mytable)\n" +
 			" │   └─ columns: [i s]\n" +
-			" └─ Table(othertable)\n" +
+			" └─ IndexedTableAccess(othertable)\n" +
+			"     ├─ index: [othertable.i2]\n" +
```

Comment:
When did we do this? I like it

Reply:
[i killed DecoratedNode one of the weeks you were out](https://github.com/dolthub/go-mysql-server/pull/1145)